app/views/recipes/public.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="recipe">
+  <h3>public recipes</h3>
+  <% @recipes.each do |recipe| %>
+    <div class='recipe_box'>
+      <div class='recipe_name'>
+       <h3><%= link_to "#{recipe.name}", recipe_path(recipe.id) %></h3>
+        <p class="recipe_user">by: <%= recipe.name %></p>
```

Comment:
- [ ] Great job so far 👏👏 However, as the previous reviewer mentioned [here](https://github.com/MargaretTechworld/recipe-app/pull/22#discussion_r1397527727) kindly make sure that the name of the recipe's owner is displaying. Currently, we have the name of the recipe in place of recipe owner's name 🐛 Kindly have a look at the screenshot attached below 📌 You can try `<%= recipe.user.inspect %>` to inspect the issue we have⛏🤓



![image](https://github.com/MargaretTechworld/recipe-app/assets/106119105/e110b3c8-1d49-4ac4-8a6e-ee97e6b4b852)